components/atoms/LanguagePill/LanguagePill.tsx

```diff
@@ -1,24 +1,39 @@
 import React from "react";
 import Image, { StaticImageData } from "next/image";
 
-import JavascriptIcon from "img/icons/👾.svg";
-import ReactIcon from "/img/icons/⚛️.svg";
-import PythonIcon from "/img/icons/🐍.svg";
-import AIIcon from "/img/icons/🤖.svg";
-import MLIcon from "/img/icons/🧠.svg";
-import RustIcon from "/img/icons/🦀.svg";
-import PhpIcon from "/img/icons/📄.svg";
-import CsharpIcon from "/img/icons/🎹.svg";
-import CIcon from "/img/icons/🥸.svg";
-import CppIcon from "/img/icons/🤓.svg";
-import TypeScriptIcon from "/img/icons/🔥.svg";
-import RubyIcon from "/img/icons/🌹.svg";
-import JavaIcon from "/img/icons/☕️.svg";
+import JavascriptIcon from "img/icons/javascript.svg";
```

Comment:
We should have one set of icons for topics. 
We have some in `img/icons` and `img/topicThumbnails`